btr-web/btr-main-app/components/individual-person/control/ControlOfDirectors.vue

```diff
@@ -40,7 +40,12 @@
     >
       <slot name="typesOfControlHelp">
         <div class="flex flex-col gap-2">
-          <p>{{ $t('helpTexts.controlOfDirectors.p1') }}</p>
+          <p><b>{{ $t('helpTexts.controlOfDirectors.directControl.title') }}</b></p>
```

Comment:
but these aren't paragraphs they are titles? To me having the <p> is confusing and giving the wrong idea for what this actually is?

Reply:
Adding or removing `<p>` tags make no visual difference. All these parts in the 'flex flex-col' div will be aligned vertically. 
Maybe I should remove the `<p>` around those title lines and leave the other text blocks as paragraphs?

For `<strong>` vs `<b>`, I think `<b>` is better because we just want to create visual hierarchy by making the title bold. `<strong>` is used to highlight important information and the screen reader will read these words with additional emphasis.

Reply:
@patrickpeinanw that's what I was thinking originally, but I think we want the additional emphasis with the screen reader right?

Reply:
if they should not be paragraphs, they should be `<span>` for text and then you can use `<strong>` where you wanted to bold it.